administrator/language/de-DE/com_joomlaupdate.ini

```diff
@@ -74,38 +93,39 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="Es sind keine Erweiterungen insta
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_SHOW_MORE_COMPATIBILITY_INFORMATION="Mehr Details"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION="Update-Informationen nicht verfügbar"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION_NOTES="Bei den folgenden Erweiterungen sind manuelle Schritte erforderlich, da diese keine mit der gewählten Zielversion kompatible Erweiterungsversion anbieten. Dies kann daran liegen, dass diese Erweiterungen nicht das <a href='https://docs.joomla.org/Special:MyLanguage/Deploying_an_Update_Server' target='_blank' rel='noopener noreferrer'>Joomla-Update-System</a> verwenden oder der Entwickler noch keine kompatible Version zur Verfügung gestellt hat."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_HELP="Weitere Informationen."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_INFOURL="Zusätzliche Informationen"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_INSTALL_SELF_UPDATE_FIRST="Zuerst muss die aktuellste Version der Joomla-Update-Komponente installiert sein, bevor der Joomla-Core aktualisiert werden kann."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_INSTALLAGAIN="Neuinstallation der Joomla-Core-Dateien"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_INSTALLED="Installierte Joomla-Version"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_INSTALLUPDATE="Update installieren"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_INSTALLUPDATE="Update"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_LATEST="Aktuellste Joomla-Version"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL="Es konnte keine Download-URL gefunden werden"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL_DESC="Ein Update für Joomla! %1$s wurde gefunden, allerdings konnte die Download-URL nicht ermittelt werden. Bitte das Update-Paket von der <a href='https://downloads.joomla.org/latest' target='_blank' rel='noopener noreferrer'>offiziellen Joomla-Downloadsite (engl.)</a> herunterladen und anschließend die Funktion „Hochladen & Aktualisieren“ nutzen."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE="Das Live-Update ist nicht verfügbar!"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE_DESC="Eine neuere Version der Joomla-Update-Komponente ist verfügbar und muss vorher installiert werden. <a class='alert-link' href='index.php?option=com_installer&view=update' target='_blank' rel='noopener noreferrer'>Hier klicken, um die Komponente zu aktualisieren</a>."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="Das System prüft gerade diese Plugins, um herauszufinden, ob während des Upgrades Probleme auftauchen könnten.<br><br>Bitte warten, bis die Prüfungen abgeschlossen sind."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Die Warnungen über möglicherweise inkompatible Plugins ignorieren und mit dem Upgrade fortfahren?"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="Das System prüft gerade diese Plugins, um herauszufinden, ob während des Updates Probleme auftauchen könnten.<br><br>Bitte warten, bis die Prüfungen abgeschlossen sind."
```

Comment:
@zero-24 Update oder Upgrade hier? Ich meine Upgrade da Major Version neu.

Reply:
In Joomla 4 wird der string ja auch bei Updates angezeigt, Update ist ok in meinen augen